tests/libnet/hotplug.go

```diff
@@ -54,7 +54,9 @@ func VerifyDynamicInterfaceChange(vmi *v1.VirtualMachineInstance, queueCount int
 	EventuallyWithOffset(1, func() []v1.VirtualMachineInstanceNetworkInterface {
 		return cleanMACAddressesFromStatus(vmiCurrentInterfaces(vmi.GetNamespace(), vmi.GetName()))
 	}, 30*time.Second).Should(
-		ConsistOf(interfaceStatusFromInterfaceNames(queueCount, secondaryNetworksNames...)))
+		ConsistOf(
+			interfaceStatusFromInterfaceNames(queueCount, vmispec.IndexInterfaceSpecByName(nonAbsentIfaces), secondaryNetworksNames...),
```

Comment:
It seems you need to pass the interfaces now and not the names.
Having both seems redundant and confusing.

How about you use `FilterInterfacesByNetworks` to collect the relevant interface and just pass them to the function here. I think you will not need the indexing as you would iterate over interface.
E.g. `FilterInterfacesByNetworks(nonAbsentIfaces, vmispec.FilterMultusNonDefaultNetworks(nonAbsentNets))`

Reply:
Done.